coderd/database/migrations/000102_workspace_agent_stats_types.up.sql

```diff
@@ -1,4 +1,4 @@
-ALTER TABLE	workspace_agent_stats ADD COLUMN connection_median_latency_ms bigint DEFAULT -1 NOT NULL;
+ALTER TABLE	workspace_agent_stats ADD COLUMN connection_median_latency_ms float DEFAULT -1 NOT NULL;
```

Comment:
Should this be done as a new migration?

Reply:
Oh my, yes! Thank goodness you noticed this :fearful: